File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/TotalSupplyOutputDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[FunctionOutput]>
    Public Class TotalSupplyOutputDTO
    
        <[Parameter]("uint256", "", 1)>
        Public Property B As BigInteger
    
    End Class

End Namespace

File: src/Nethereum.StandardTokenEIP20/DTOs/TotalSupplyOutputDTO.cs
using System.Numerics;
using Nethereum.ABI.FunctionEncoding.Attributes;
using Nethereum.Contracts;

namespace Nethereum.StandardTokenEIP20.DTOs
{
    [FunctionOutput]
    public class TotalSupplyOutputDTO: IFunctionOutputDTO
    {
        [Parameter("uint256", "", 1)]
        public BigInteger B {get; set;}
    }
}
